<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Profile name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTextEdit" name="textEditName">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTextEdit" name="textEditIWAD">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>IWAD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="listFiles"/>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QPushButton" name="buttonMoveUp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="buttonMoveDown">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Down</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="buttonAddFile">
           <property name="text">
            <string>Add file...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="buttonRemove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEditName</sender>
   <signal>textChanged()</signal>
   <receiver>Form</receiver>
   <slot>change_name()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEditIWAD</sender>
   <signal>textChanged()</signal>
   <receiver>Form</receiver>
   <slot>change_iwad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddFile</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>add_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemove</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>remove_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonMoveUp</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>move_file_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonMoveDown</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>move_file_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accept()</slot>
  <slot>rejected()</slot>
  <slot>add_file()</slot>
  <slot>change_name()</slot>
  <slot>change_iwad()</slot>
  <slot>remove_file()</slot>
  <slot>move_file_up()</slot>
  <slot>move_file_down()</slot>
 </slots>
</ui>
